libraries/src/TUF/DatabaseStorage.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Joomla! Content Management System
+ *
+ * @copyright  (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\CMS\TUF;
+
+use Joomla\CMS\Table\Table;
+use Joomla\CMS\Table\Tuf;
+use Joomla\Database\DatabaseDriver;
+use Tuf\Metadata\StorageBase;
+
+\defined('JPATH_PLATFORM') or die;
+
+/**
+ * @since  __DEPLOY_VERSION__
+ */
+class DatabaseStorage extends StorageBase
+{
+    const METADATA_COLUMNS = ["root", "targets", "snapshot", "timestamp", "mirrors"];
+
+    /**
+     * The Tuf table object
+     *
+     * @var Table
+     */
+    protected $table;
+
+    protected $container = array();
```

Comment:
Variable comment needed
```suggestion
    protected $container = [];
```